src/lgr_advanced/lgr_editor/forms/importer.py

```diff
@@ -27,11 +28,7 @@ class ImportLGRForm(forms.Form):
                            help_text=f"{_('If you select more than one file, this will create a LGR set.')} "
                                      f"{FILE_FIELD_ENCODING_HELP}",
                            widget=forms.ClearableFileInput(attrs={'multiple': True}))
-    validating_repertoire = forms.ChoiceField(label=_("Validating repertoire"),
-                                              help_text=_('Code points will be limited to the selected repertoire'),
-                                              required=False,
-                                              choices=(('', ''),) + VALIDATING_REPERTOIRES,
-                                              initial=DEFAULT_VALIDATING_REPERTOIRE)
+    validating_repertoire = forms.ModelChoiceField(queryset=RzLgr.objects.all(), empty_label=None, to_field_name='name')
```

Comment:
I would add a note (TODO) to say that this updates limits the choices to RZ LGRs while we add other types of validating repertoire before, but this is because future updates will add the other repertoires as well